label_studio/fsm/queryset_mixins.py

```diff
@@ -53,25 +53,38 @@ def get_queryset(self):
                 return TaskQuerySet(self.model, using=self._db)
 
             def with_state(self):
-                return self.get_queryset().annotate_fsm_state()
+                return self.get_queryset().with_state()
 
-        # Usage
+        # Usage - both approaches work identically
         tasks = Task.objects.with_state().filter(project=project)
+        # Or chain it after filters
+        tasks = Task.objects.filter(project=project).with_state()
+
         for task in tasks:
             print(f"Task {task.id}: {task.current_state}")  # No additional queries!
     """
 
-    def annotate_fsm_state(self):
+    def with_state(self):
```

Comment:
looks like this is purely a rename, no functional changes? Moving the mixin classes to top level is the only functional change?

Reply:
Exactly, just a bunch of find and replace :)